cherrypy/_cpreqbody.py

```diff
@@ -745,7 +765,13 @@ def __init__(self, fp, length, maxbytes, bufsize=DEFAULT_BUFFER_SIZE,
         # Wrap our fp in a buffer so peek() works
         self.fp = fp
         self.length = length
-        self.maxbytes = maxbytes
+        if isinstance(maxbytes, tuple):
```

Comment:
After thinking this through, I've recalled that there's some places in config, which allow setting callable instead of scalar as a value. We should probably allow setting `maxbytes` to callable and test it like:
```python
try:
    self.maxbytes = maxbytes()
except TypeError:
    self.maxbytes = maxbytes
```